tests/test_secure_random.py

```diff
@@ -0,0 +1,50 @@
+import unittest
+
+import tensorflow as tf
+import numpy as np
+from tf_encrypted.operations.secure_random import secure_random
+
+
+class TestSecureRandom(unittest.TestCase):
+
+    def test_wrapper(self):
+        expected = [[6610, 5100, 676],
+                    [6111, 9801, 5407],
+                    [9678, 7188, 8280]]
+
+        with tf.Session():
+            output = secure_random([3, 3], maxval=10000).eval()
+
+            np.testing.assert_array_equal(output, expected)
+
+    def test_min_val(self):
+        expected = [[3220, 200, -8648],
+                    [2223, 9603, 815],
+                    [9356, 4377, 6561]]
+
+        with tf.Session():
+            output = secure_random([3, 3], minval=-10000, maxval=10000).eval()
```

Comment:
uh, this looks dangerous! :D it will used a fixed seed if none is specified? if so we should definitely change that (potentially requiring a seed to always be specified)!